packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -421,6 +426,17 @@ export class WorkflowsController {
 		return { ...workflow, scopes, checksum };
 	}
 
+	@Get('/:workflowId/exists')
+	async exists(req: WorkflowRequest.Get) {
+		const { workflowId } = req.params;
+
+		const workflow = await this.workflowFinderService.findWorkflowForUser(workflowId, req.user, [
```

Comment:
this is misleading. `findWorkflowForUser` could return null but a workflow with that id could still exist. 
This also returns a lot more than what's needed  for this context.
What we need is to find a workflow and only "select" its id to return. Maybe the same `existsBy` used above would suffice